lib/libutils/ext/include/util.h

```diff
@@ -188,7 +188,33 @@ static inline void reg_pair_from_64(uint64_t val, uint32_t *reg0,
 	*reg1 = low32_from_64(val);
 }
 
-/* Get and set bit fields  */
+/**
+ * Functions to get and set bit fields in a 32/64-bit register.
```

Comment:
I would replace `register` with `value` (several occurrences) . These helper functions are not specifically designed for values read from/written to hardware registers, they apply to any 32bit/64bit values.

Reply:
I agree with you these functions could be use to manipulate 32/64 bit value but they are intended for use on registers that why the first argument is named "reg". So I think it is better keep the register name to avoid confusion.

Reply:
> ... they are intended for use on registers ...

I don't agree. One can use them for some other purpose, e.g. extracting bits from a DT config argument or any memory cell.

Reply:
OK I propose to simply replace register by bitfield

Reply:
s/register/value/?